bedrock-devnet/account_abstraction/__init__.py

```diff
@@ -0,0 +1,125 @@
+import argparse
```

Comment:
Not saying it's the wrong approach, but did we consider consolidating this with the other script, say as a parameter? I'm a bit worried about upstream drift breaking this script and having to manually identify and pull in changes.

Reply:
Yes, this script is a standalone script. It essentially deploys all AA smart contracts and brings up the services using `docker-compose-side.yml`. If upstream changes the local network or deployment key, etc., then we need to modify this. Otherwise, everything should work fine.